tests/python/FileTransformTest.py

```diff
@@ -0,0 +1,207 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+import unittest
+
+import PyOpenColorIO as OCIO
+from UnitTestUtils import TEST_NAMES
+
+
+class FileTransformTest(unittest.TestCase):
+    TEST_DIRECTION = OCIO.TRANSFORM_DIR_FORWARD
+    TEST_INTERPOLATION = OCIO.Interpolation.INTERP_LINEAR
+    TEST_ID = 'sample file'
+    TEST_SRC = 'foo'
+    TEST_DST = 'bar'
+    DEFAULT_FORMATS = [('flame', '3dl'),
```

Comment:
Recommend not hard coding these value (`DEFAULT_FORMATS`)s in the Python test.

Reply:
If you don't mind elaborating on what you think would be a better way to check the default `getFormats()` function. I created that list based off of a loop and so I just want to ensure future `getFormats()` call equal that.

Reply:
Yeah I think you're right. I was hesitating to add an additional place where these are hard coded since they all must be maintained, but this is probably the most useful test regardless.